demomultistoreform/src/Controller/DemoMultistoreController.php

```diff
@@ -102,22 +123,21 @@ public function edit(Request $request, int $contentBlockId): Response
         ]);
     }
 
-    public function delete(Request $request, int $contentBlockId): Response
+    public function delete(
+        int $contentBlockId,
+        EntityManagerInterface $entityManager,
+        Context $shopContext
```

Comment:
We shouldn't use this one anymore (which makes me think I should deprecated it), instead we now have the new `ShopContext` available, and you don't even need to inject it because it's accessible via the `PrestaShopAdminController::getShopContext()`